src/ca11y/index.js

```diff
@@ -200,6 +222,7 @@ class Ca11y {
   }
 
   setUI(el) {
+    el.setAttribute('type', 'text') // in case this is [type="date"]
```

Comment:
Yeah, needed in case source markup is `<input type="date">` and  the option `useNative` is false. Otherwise, the native date input styling and features will be present instead of the text input.